### ISSUE TITLE
Fix intermittent VSM download test cases

### ISSUE THREAD

**Issue by scovetta** [microsoft/OSSGadget#165]
The VSM download tests are failing in CI builds, but not locally. It seems to be because on CI, different folders are created. We should get to the bottom of it.

**Comment by scovetta** [comment]
I'm suspecting this is actually a bug in RecursiveExtractor. When testing against a VSM file on Linux *only*, a license text file is "extracted" into another file, which is really strange.

```
Linux:
 /tmp/xfbzlkqg.xnw/vsm-ms-vscode.PowerShell@2020.6.0-Microsoft.VisualStudio.Services.Content.License/vsm-ms-vscode.PowerShell@2020.6.0-Microsoft.VisualStudio.Services.Content

Windows:
C:\Users\miscovet\AppData\Local\Temp\0tzupwfh.vzg\vsm-ms-vscode.PowerShell@2020.6.0-Microsoft.VisualStudio.Services.Content.License
```

**Comment by gfs** [comment]
Can you upload a sample VSM file here for me to test with RE?

What is VSM?

**Comment by scovetta** [comment]
VSM = Visual Studio Marketplace

You can grab it with `oss-download pkg:vsm/vs-code/PowerShell`. 

Thanks Gabe!

**Comment by scovetta** [comment]
This should be fixed in #168
